src/collection-preferences/content-display/index.tsx

```diff
@@ -134,11 +134,14 @@ export default function ContentDisplayPreference({
           itemDefinition={{ id: item => item.id, label: item => item.label }}
           onItemsChange={({ detail }) => onChange(detail.items)}
           disableReorder={columnFilteringText.trim().length > 0}
-          renderItem={({ ref, item, style, className, dragHandleProps }) => {
+          renderItem={({ ref, item, style, className, dragHandleProps, isDragGhost }) => {
             className = clsx(className, getOptionClassName());
             const content = (
               <ContentDisplayOption ref={ref} option={item} onToggle={onToggle} dragHandleProps={dragHandleProps} />
             );
+            if (isDragGhost) {
+              return content;
+            }
```

Comment:
Approved because it is a hotfix, but this begs the question – should there be any `aria-hidden` or other attributes set for this content?

This exacerbates when distributing `SortableArea` as a public component. Every team is supposed to not forget about this use-case, making it prone to issues

Reply:
This code is a small revert to our current approved implementation of the collection preferences content display, so I expect it to be alright. The issue will styles should be solvable in a way that depending on the isDragGhost is not needed.

Reply:
Ok